linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/oval/shared.xml

```diff
@@ -27,6 +27,10 @@
         {{% endif %}}
         <criterion comment="Check ClientAliveCountMax in /etc/ssh/sshd_config"
         test_ref="test_sshd_clientalivecountmax" />
+        {{%- if config_is_distributed == "true" %}}
+        <criterion comment="Check ClientAliveCountMax in /etc/ssh/sshd_config.d/ files"
```

Comment:
`/etc/ssh/sshd_config` and `/etc/ssh/sshd_config.d/` criterion needs to be wrapped in `criteria` element with `operator="OR"`